SeleniumSolarTechnology/SolarTechnology/Tests/CartTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using SolarTechnology.Pages;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SolarTechnology.Tests
+{
+    public class CartTests : UITestFixture
+    {
+        [Test]
+        public void VerifyAddToCartAndRemoveFromCartProductItem()
+        {
+            var catalogPage = new CatalogPage(_driver);
+            var homePage = new HomePage(_driver);
+            var cartPage = new CartPage(_driver);
+
+            homePage.Open();
+            homePage.OpenCableAndSwitching();
+                catalogPage.AddProductItemToCart(1);
```

Comment:
if next line is a new statement, it should not be moved to the right. Remove 4 spaces before "catalogPage".
The same for next two lines - all these lines should be aligned the same.
Additional 4 spaces are added when we move part of single line to the new line. For example:
var someFilteredList = someList.Where(element => element.SomeField > 10).Select(element => element.SomeField);
can be written like
```
var someFilteredList = someList
    .Where(element => element.SomeField > 10)
    .Select(element => element.SomeField);
```

Reply:
ok, understood, fixed.